views/vwLearner/homeRevision.hbs

```diff
@@ -25,7 +44,24 @@
 {{/section}}
 
 <div class="container-fluid">
-  <div class="chart pt-3">
+  <div class="instruction">
+    <i class="bi bi-question-circle"></i>
+  </div>
+
+  <div class="chart">
+    <div class="hide">
+      <p class="instruction-text">
+        You will learn new words and they will be divided into 5 levels. <br>
+        <b>Level 1</b> includes the new words and they will be tested daily. <br>
+        <b>Level 2</b> includes words from level 1 that you answer correctly in the review test, and these words will appear again in
+        the review test after 3 days. <br>
+        <b>Level 3</b> includes words from level 2 that you answer correctly, and they will appear again in the review test after 5
+        days. <br>
+        <b>Level 4</b> includes words from level 3 that you answer correctly, and they will appear again in the review test after 7
+        days. <br>
+        <b>Level 5</b>'s words will not appear in the review test as you have completely learned them by heart. <br>
+      </p>
```

Comment:
b có thể tạo 1 file `messages.js` rồi tạo các constant trong đó kiểu như `const instructions = {"title": "You will learn new words and they will be divided into 5 levels.", "level1": ...}`

Reply:
ok nhưng ko ô